src/components/SearchForm/SearchForm.scss

```diff
@@ -0,0 +1,25 @@
+.SearchForm {
```

Comment:
Please add some gap between the input and the button. Also, ensure they are aligned vertically at the center.

![image](https://github.com/mradomski-epam/reactjs-mentoring/assets/118353174/e640564d-450f-4135-ab86-e0cc5586746f)